tests/tck/utils/assertions.py

```diff
@@ -1,10 +1,76 @@
+from numbers import Number
+from redisgraph import Node, Edge
 def is_number_tryexcept(s):
-    """ Returns True is string is a number. """
+    """ Returns True if string is a number. """
+    if not isinstance(s, (Number, basestring)):
+        return False
     try:
         float(s)
         return True
     except ValueError:
         return False
+    
+def prepareActualValue(actualValue):
+        actualValue = prepareString(actualValue)
+        actualValue = stringToNumeric(actualValue)
+        actualValue = nodeToString(actualValue)
+        actualValue = edgeToString(actualValue)
+        return actualValue
+
+def prepareExpectedValue(expectedValue):
+    # string preparation
+    expectedValue = prepareString(expectedValue)
+    if expectedValue == "true":
+        expectedValue = True
+    if expectedValue == "false":
+        expectedValue = False
+    if expectedValue == "null":
+        expectedValue = None
+    expectedValue = stringToNumeric(expectedValue)
```

Comment:
Do we really need to call stringToNumeric if we entered any of the conditions above?

Reply:
done